tests/containers/bci_test.pm

```diff
@@ -92,6 +92,8 @@ sub run {
 
     record_info('Run', "Starting the tests for the following environments:\n$test_envs");
     my $bci_dir = fileparse($bci_tests_repo, qr/\.[^.]*/);
+    # sometimes ppc64le are still running type_string 'cd' and fail to run cd BCI-tests
```

Comment:
This comment suggest that the sleep should be in the next line. So either the comment is misleading or the sleep needs to be one line down.